### PR TITLE
added liveness probe commands to use mongo instead of mongosh 

### PR DESCRIPTION
**Pull request by rosesyrett**

### CODE REVIEW

File: helm/diffcalc-api/values.yaml [hunk 1]
```diff
@@ -103,6 +103,18 @@ mongodb:
   persistence:
     existingClaim: diffcalc-mongo-volume
     createNew: false
+  livenessProbe:
```
Comment (callumforrester) [inline helm/diffcalc-api/values.yaml:106]: Looking at the template values, I think you need `customLivenessProbe`

https://github.com/bitnami/charts/blob/main/bitnami/mongodb/values.yaml#L536

**Review by callumforrester — CHANGES_REQUESTED** [review]
Should we also do the same for `readinessProbe`?

File: helm/diffcalc-api/values.yaml (hunk 1 above)
Comment (rosesyrett) [reply helm/diffcalc-api/values.yaml:106]: All done now :+1:

**Review by callumforrester — APPROVED** [review]
Looks good to me, although I haven't tested it

**Pull request merged**